Drv/TcpClient/test/ut/TcpClientTester.cpp

```diff
@@ -142,9 +157,12 @@ void TcpClientTester ::test_advanced_reconnect() {
   {
     this->pushFromPortEntry_recv(recvBuffer, recvStatus);
     // Make sure we can get to unblocking the spinner
-    EXPECT_EQ(m_data_buffer.getSize(), recvBuffer.getSize()) << "Invalid transmission size";
-    Drv::Test::validate_random_buffer(m_data_buffer, recvBuffer.getData());
-    m_spinner = true;
+    if(recvStatus == Drv::RecvStatus::RECV_OK){
```

Comment:
Was this added to fix an issue, or patch one of the threading failures we had?

Reply:
Ah you're right, I should have realized the thread issue here. Removed those now. Ty ty